pages/docs/manual/latest/bind-to-js-function.mdx

```diff
@@ -421,3 +421,34 @@ Currently 4 directives are supported: `null_to_opt`, `undefined_to_opt`, `nullab
 <!-- When the return type is unit: the compiler will append its return value with an OCaml unit literal to make sure it does return unit. Its main purpose is to make the user consume FFI in idiomatic OCaml code, the cost is very very small and the compiler will do smart optimizations to remove it when the returned value is not used (mostly likely). -->
 
 `identity` will make sure that compiler will do nothing about the returned value. It is rarely used, but introduced here for debugging purpose.
+
+## Tagged template functions
+
+**Since 11.1**
+
+**Experimental** You can easily bind to JS tagged template functions.
+All you need to do is defining a binding to a function that has two arrays as arguments, 
+the first one being an array of strings and the second can be an array of anything.
```

Comment:
Some suggestions:
- Link to the relevant MDN article: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates
- Highlight that this form/signature is because that's how it works in JS

My reasoning is that most people will likely have heard about tagged templates, but not necessarily know anything about the internals. So I think it's important to highlight that this is all about interop, and that this is the way that things work in JS, that's why they look like they look.